examples/stroke/random.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+        "context"
+        "flag"
+        "fmt"
+        "log"
+        "os"
+	"math/rand"
+        "time"
+
+        "github.com/funjack/golaunch"
+)
+
+var (
+        start    = flag.Int("start", 5, "start position")
+        end      = flag.Int("end", 95, "end position")
+        speed    = flag.Int("speed", 30, "speed")
+        interval = flag.Int("interval", 300, "interval in milliseconds")
+)
+
+func main() {
+	// TODO: keyboard controls for (p)ause, (k)eep doing that, and (o)max speed/range
+
+	flag.Parse()
+
+        launchContext := context.Background()
+
+        l := golaunch.NewLaunch()
+	log.Printf("Connecting...")
+        l.HandleDisconnect(func() {
+                os.Exit(0)
+        })
+        ctx, cancel := context.WithTimeout(launchContext, time.Second*30)
+        err := l.Connect(ctx)
+        cancel()
+        if err != nil {
+                log.Fatal(err)
+        }
+	log.Printf("Connected to Launch")
+
+        ticker := time.Tick(time.Duration(*interval) * time.Millisecond)
+
+        go func() {
+
+	}()
+
+	for {
+		rand.Seed(time.Now().UTC().UnixNano())
+		var count = rand.Intn(120)
+		var start = rand.Intn(30)
+		var end = rand.Intn(100)
+		var speed = rand.Intn(100)
+		var interval = rand.Intn(350)
+
+		// you can't feel much in <150ms so increase if necessary
+		if interval < 150 {
+			interval = 150
+		}
+		ticker = time.Tick(time.Duration(interval) * time.Millisecond)
+
+		// the slow edging crowd should comment this out
```

Comment:
Same thing as going to fast. The official apps never go below 20%, we don't know if this will cause any damage. We do know that going very slow will crash the Launch. So if/when advising people to use slower speeds at least point out the risks.